doc/tutorials/01_Param.ipynb

```diff
@@ -0,0 +1,338 @@
+   "source": [
+    "text_input.value = 3.14"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a0d99de-9c48-4e68-89f9-2d911859e440",
+   "metadata": {},
+   "source": [
+    "### Class vs. Instance Parameters\n",
+    "\n",
+    "Parameters are defined on classes and when a `Parameterized` is created the parameters defined on it will be initialized. This means that the parameter default value and any validation related attributes defined on the parameter can be modified and all instances created thereafter will reflect these changes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbb7b05d-adb6-443a-a163-5ef8ccefb946",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#pn.widgets.TextInput.param.styles.default = {'border': '1px solid black', 'padding': '5px'}\n",
+    "\n",
+    "pn.widgets.TextInput()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d41b900a-6527-4fbd-98b5-dcfbd7f92b4b",
+   "metadata": {},
+   "source": [
+    "## Parameters as references\n",
+    "\n",
+    "Above we mentioned that `Parameter` objects can act as proxies or references for the underlying Parameter value. This is a powerful feature when trying to declare interactivity. Let us see what we mean by that with a simple example again based on the `TextInput` widget:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71308903-925f-4e98-aafc-131b42d21e46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_in = pn.widgets.TextInput(value='Hello world!')\n",
+    "text_out = pn.pane.Markdown(text_in.param.value)\n",
+    "\n",
+    "pn.Column(text_in, text_out)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f92f1dd5-9d54-4660-9c5c-884ae2127a2b",
+   "metadata": {},
+   "source": [
+    "Note how we were able to pass the `value` parameter to the `Markdown` pane. If you attempt to type into the `TextInput` you'll notice that this is no automatically reflected by the Markdown output.\n",
+    "\n",
+    "This also works when using it for other parameters, e.g. we can add a switch to toggle the visibility of some component:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4af0c00b-1e90-47c6-823a-692bb023fb87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "visible = pn.widgets.Switch(value=True)\n",
+    "\n",
+    "pn.Row(visible, pn.pane.Markdown('Hello World!', visible=visible))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ee3d80f-19f8-4643-afc8-f8ba80ec4267",
+   "metadata": {},
+   "source": [
+    "Notice that we passed in the widget object directly instead of the `.param.value`. This is possible because widgets are treated as a proxy of their `value` parameter just like a `Parameter` is treated as a proxy for current value."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55f2a3fe-771f-4aff-b2eb-b107e373b735",
+   "metadata": {},
+   "source": [
+    "### Indirect Mapping\n",
```

Comment:
I thought that `pn.bind` and Parameters as reference belonged to the *Interactivity* guide.

Reply:
Parameters as references and pn.bind (and eventually expressions) is definitely core Param material. The Interactivity guide will go significantly beyond these basic concepts. It may turn out that this is not good material as beginner docs, we are definitely aiming to deliver quite advanced material that goes significantly beyond "basics".